### ISSUE TITLE
If no-rfc5780 is set, alt-listening-port and alt-tls-listening-port are ignored

### ISSUE THREAD

**Issue by al-com** [coturn/coturn#1038]
# Version
Turnserver version 4.6.0

# Question
If `no-rfc5780` is set in the turnserver.conf the values `alt-listening-port` and `alt-tls-listening-port` are ignored. Is this working as designed?

The description is unclear in that regard:
> alt-listening-port
>```
># Alternative listening port for UDP and TCP listeners;
># default (or zero) value means "listening port plus one".
># This is needed for RFC 5780 support
># (STUN extension specs, NAT behavior discovery). The TURN Server
># supports RFC 5780 only if it is started with more than one
># listening IP address of the same family (IPv4 or IPv6).
># are listening to that endpoint only for "symmetry".
>```

> alt-tls-listening-port
>```
> # Alternative listening port for TLS and DTLS protocols.
># Default (or zero) value means "TLS listening port plus one".
>```

**Comment by ggarber** [comment]
Yes, only rfc5780 requires those additional ports so if it is disabled it is normal that those don't take any effect.

Is there any problem with that?

**Comment by jbg** [comment]
You can use `listening-port` and `tls-listening-port`. Despite the names, coturn actually accepts both non-TLS and TLS connections on both ports. ref: https://github.com/coturn/coturn/blob/master/README.turnserver#L329

**Comment by olmari** [comment]
@ggarber

> Is there any problem with that?

Can I point multiple listening ports? I'm basically wanting to have both 2 of the "traditional" ports, and then 443 also, so basically 3 separate ports. So I kind of do have problem with this particularly, irrelevant to RFC 5780 specifically.

**Comment by eakraly** [comment]
2.5 years late....
Use `--aux-server=listening_IP:Port` to listen on additional ports
For example:
```
turnserver -L 10.0.0.2 \
  --aux-server=10.0.0.2:3480 \
  --aux-server=10.0.0.2:3481 \
  --aux-server='[2001:db8::1]:3480'
...
```
